### ISSUE TITLE
Cannot import Astar address with ledger astar app and got CLA not support error

### ISSUE THREAD

**Issue by EV6999** [polkadot-js/apps#10823]
Tried to import Astar address on polkadot-js extension on chrome with Ledger Astar app and got CLA not support error.

**Comment by TarikGul** [comment]
Yes, this is known, and I am actually in the process of getting this sorted in both the extension and apps. It should be ready and in soon.
